src/components/cards/wishlistCard.js

```diff
@@ -0,0 +1,26 @@
+import './index.css'
+import { GoX } from "react-icons/go";
+const WishlistCard=({productObj})=>{
+    const{price,imageUrl,title,description,productRating}=productObj
```

Comment:
you could have destructured this product obj in the function parameter itself no need to create a constant for that.